cli/config_file.rs

```diff
@@ -293,7 +294,10 @@ impl ConfigFile {
   }
 
   pub fn new(text: &str, path: &Path) -> Result<Self, AnyError> {
-    let jsonc = jsonc_parser::parse_to_serde_value(text)?.unwrap();
+    let jsonc = match jsonc_parser::parse_to_serde_value(text) {
+      Ok(Some(value)) if value.is_object() => value,
+      _ => return Err(anyhow!("Unable to parse config file JSON")),
```

Comment:
Would be better to provide the path as as part of the error message.

Reply:
We might also be consistent with import_map error. I used the same error message.
If we plan to update this one, We may also update import_map

Reply:
Yeah, we should improve that. But that would be a seperate PR/issue. Opened #10768.